index.html

```diff
@@ -49,13 +49,13 @@ <h1>
 <p>
   These links will take you to the Azure IoT Hub SDKs for Java API reference documentation 
   <ul>
-    <li><a href="device">Azure IoT Hub Device SDK for Java API reference</a></li>
-    <li><a href="service">Azure IoT Hub Service SDK for Java API reference</a></li>
-    <li><a href="provisioning/provisioning-device-client">Azure IoT Hub Provisioning Dervice SDK for Java API reference</a></li>
-    <li><a href="provisioning/provisioning-service-client">Azure IoT Hub Provisioning Service SDK for Java API reference</a></li>
-    <li><a href="provisioning/security/security-provider">Azure IoT Hub Security Provider SDK for Java API reference</a></li>
-    <li><a href="provisioning/security/tpm-provider">Azure IoT Hub TPM Provider SDK for Java API reference</a></li>
-    <li><a href="provisioning/security/x509-provider">Azure IoT Hub X509 Provider SDK for Java API reference</a></li>
+    <li><a href="master/device">Azure IoT Hub Device SDK for Java API reference</a></li>
```

Comment:
Files below are just the content moved into this folder. It might be easier to checkout the branch and see that instead of the PR.